src/cli.coffee

```diff
@@ -19,6 +19,8 @@ if args[0] is "install"
           sudo cp #{__dirname}/meh.resolver /etc/resolver/dev
           sudo cp #{__dirname}/meh.firewall.plist /Library/LaunchDaemons/meh.firewall.plist
           sudo launchctl load -w /Library/LaunchDaemons/meh.firewall.plist
+          cp #{__dirname}/meh.mehserve.plist ~/Library/LaunchAgents/meh.mehserve.plist
+          launchctl load ~/Library/LaunchAgents/meh.mehserve.plist
```

Comment:
Since you've added this, the comment about pm2 no longer makes sense (if you follow these steps then you don't need pm2), so would you mind splitting this and the pm2 comment into their own section basically stating that if mehserve isn't running then DNS resolution will be slowed, and you can run mehserve in the background either either using pm2, or by installing it as a LaunchDaemon, or via some other means.

Thanks! 👍